run-cnf-suites.sh

```diff
@@ -66,5 +66,12 @@ if [[ ! -f "/proc/1/cgroup" ]] || grep -q init\.scope /proc/1/cgroup; then
 	cd ..
 fi
 
+if [[ -z "${TNF_PARTNER_SRC_DIR}" ]]; then
+	echo "env var \"TNF_PARTNER_SRC_DIR\" not set, please set it before running this command!"
+	exit 1
+fi
+
+make -C $TNF_PARTNER_SRC_DIR install-partner-pods
```

Comment:
This is breaking the container TNF image. Make and other tools are removed before the image is finished. Need to test and update testing using container image

Reply:
@edcdavid @pkliczewski 
"Need to test and update testing using container image"
You mean adding a test to the CI that uses the the image built in the previous tests.